### DISCUSSION TITLE
tab activation on button press

### DISCUSSION THREAD

**Discussion by klikr** [piroor/treestyletab#3021] [Q&A] [answered]
Hi,

after starting up FF I often shift around FF tabs in trees. When I press and hold the mouse button on a (inactive, currently unloaded) tab to drag it to its new location, the tab is activated and the content loads, even if another tab is currently active. Can this behavior be somehow modified to not activate the tab on drag&drop?

**Comment by piroor** [comment, marked as answer]
What about the helper addon [Move unloaded tabs for Tree Style Tab](https://addons.mozilla.org/firefox/addon/move-unloaded-tabs-for-tst/) ?

**Reply by klikr** [reply]
wow, exactly what I was looking for. Thanks bunch, mate!
